cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -304,6 +309,32 @@ private DateTime queryDate(String propertyName, DateTime defaultValue) {
         return d;
     }
 
+    /**
+     * Parse a date in ISO format. If the offset is not present at the end (ie. no "Z" nor "+xx:xx" or "+xxxx"), it is
```

Comment:
Just wondering why you did need this... joda was doing it automatically?

Reply:
agree, there is something to investigate here

Reply:
The difference with Joda is that, while there was a unique class DateTime in Joda, there is now multiple classes in java.time : LocalDateTime, ZonedDateTime, OffsetDateTime.
With Joda, if you parsed "yyyy-MM-dd'T'HH:mm:ss.SSSS+xx:xx" or "yyyy-MM-dd'T'HH:mm:ss.SSSS", you would get the same type of data: a DateTime.
With Java.Time, if you parse "yyyy-MM-dd'T'HH:mm:ss.SSSS+xx:xx", you can get a ZonedDateTime (date + time + offset to UTC) while if you parse "yyyy-MM-dd'T'HH:mm:ss.SSSS", you get only a LocalDateTime (date + time), which has less information. If you want to convert the LocalDateTime to a ZonedDateTime, you have to specify which offset to UTC you are assuming to have.

Reply:
To avoid this annoying if, you'd better use parseBest, which is meant exactly for that purpose:
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/format/DateTimeFormatter.html#parseBest(java.lang.CharSequence,java.time.temporal.TemporalQuery...)

Reply:
done